tmva/sofie/test/CMakeLists.txt

```diff
@@ -37,8 +37,8 @@ target_include_directories(emitFromONNX PRIVATE
 )
 
 target_link_libraries(emitFromONNX ${Protobuf_LIBRARIES} ROOTTMVASofie)
-set_target_properties(emitFromONNX PROPERTIES
-  POSITION_INDEPENDENT_CODE TRUE)
+set_target_properties(emitFromONNX PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
+target_compile_options(emitFromONNX PRIVATE -Wno-unused-parameter -Wno-array-bounds)
```

Comment:
Can you add `# not needed with protobuf version ...` so we know we can kick this out in a couple of years? Same to the other two occurrences, please.